akka-remote/src/test/scala/akka/remote/MessageLoggingSpec.scala

```diff
@@ -52,8 +52,9 @@ class ClassicMessageLoggingSpec extends MessageLoggingSpec(config(false))
 
 abstract class MessageLoggingSpec(config: Config) extends AkkaSpec(config) with ImplicitSender {
 
-  val remoteSystem = ActorSystem("remote-sys", config)
+  val remoteSystem = ActorSystem("remote-sys", ConfigFactory.load(config))
```

Comment:
This should not be needed, the ActorSystem does that.
I use `sbt -D...` "all the time", why wouldn't that work here?

Reply:
ah, that is done by AkkaSpec, my bad